<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Dialog::DlgConfigurationWizard</class>
 <widget class="QDialog" name="Gui::Dialog::DlgConfigurationWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>454</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>418</y>
     <width>541</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="_3">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="logo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Teste teste teste</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="tabWidgetStack_3">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>506</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Welcome</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>User System:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>GUI Language</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Mouse behaviour</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_ViewSystem">
             <item>
              <property name="text">
               <string>Standard (mm/kg/s/degree)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>MKS (m/kg/s/degree)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>US customary (in/lb)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Imperial decimal (in/lb)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Building Euro (cm/m²/m³)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Building US (ft-in/sqft/cuft)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Metric small parts &amp; CNC(mm, mm/min)</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="Languages"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboNavigationStyle">
             <property name="currentIndex">
              <number>-1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
